staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -237,10 +247,9 @@ func UpdateInflightRequestMetrics(nonmutating, mutating int) {
 	currentInflightRequests.WithLabelValues(MutatingKind).Set(float64(mutating))
 }
 
-// Record records a single request to the standard metrics endpoints. For use by handlers that perform their own
-// processing. All API paths should use InstrumentRouteFunc implicitly. Use this instead of MonitorRequest if
-// you already have a RequestInfo object.
-func Record(req *http.Request, requestInfo *request.RequestInfo, component, contentType string, code int, responseSizeInBytes int, elapsed time.Duration) {
+// RecordRequestError records the occurrence of a request error during apiserver handling (e.g. timeouts,
```

Comment:
`RecordRequestError records that the request was terminated early as part of a resource preservation or apiserver self-defense mechanism (e.g. ...`